wiki/Tournaments/OWC/2015/de.md

```diff
@@ -1,215 +1,480 @@
+   - Jeder Spieler wird als potenzielles Teammitglied in Betracht gezogen. Die Flagge im Profil ist ausschlaggebend.
+   - Um erfolgreich akzeptiert zu werden, muss dein globaler Rang in osu! mindestens \#5000 oder besser sein.
+   - Um erfolgreich akzeptiert zu werden, solltest du sichergehen, dass du nicht die [osu! Community-Regeln](/wiki/Rules) in den letzten 12 Monaten verletzt hast.
 3. Alle erfolgreich zusammengestellten Teams werden nach der Registrierungsphase veröffentlicht.
 4. Abgelehnte Spieler können diese Entscheidung anfechten, indem sie [tournaments@ppy.sh](mailto:tournaments@ppy.sh) kontaktieren.
-5. Map Wähler dürfen nicht am Turnier als Spieler teilnehmen.
+5. Beatmap-Wähler dürfen nicht am Turnier als Spieler teilnehmen.
 
 ### Etappenanweisungen
 
 1. In der ersten Etappe (Gruppenphase) werden alle Teams in 8 Gruppen mit jeweils 4 Teams aufgeteilt.
-
 2. Die Teams werden gegen alle anderen Teams in derselben Gruppe antreten.
-
 3. Rankings werden für jede Gruppe durch eine Sortierung von den Ergebnissen anhand der Performance jedes Teams an den folgenden Prioritäten ermittelt:
-   1. Am meisten gewonnene Partien.
-   2. Hat ein größeres `{(Anzahl an gewonnenen Maps) - (Anzahl an verlorenen Maps)}`.
-   3. Am meisten gewonnene Maps.
-   4. Hat ein größeres `∑{(gesamtes Ergebnisdifferenz) / (maximales Ergebnis)}`.
-   5. Gewinner von Rückspielen.
-
-4. Die besten 2 Teams in der Gruppe kommen in die Double-Elimination Etappen.
-
-5. Die folgenden Etappen sind Double-Elimination Etappen. Dies bedeutet, dass der Gewinner normal in die nächste Etappe kommt und das andere Team in die Verliereretappe kommt.
-
+   - Am meisten gewonnene Partien.
+   - Hat ein größeres `{(Anzahl an gewonnenen Maps) - (Anzahl an verlorenen Maps)}`.
+   - Am meisten gewonnene Maps.
+   - Hat ein größeres `∑{(gesamtes Ergebnisdifferenz) / (maximales Ergebnis)}`.
+   - Gewinner von Rückspielen.
+4. Die besten 2 Teams in der Gruppe kommen in die Doppeleleminierungsetappen.
+5. Die folgenden Etappen sind Doppeleleminierungsetappen. Dies bedeutet, dass der Gewinner normal in die nächste Etappe kommt und das andere Team in die Verliereretappe kommt.
 6. Basierend auf [dieser Grafik](https://puu.sh/bUq5V/f1066103b0.png) teilen sich die Etappen wie folgt auf:
 
-   | Stage | Match ID |
-   | :-- | :-- |
-   | Achtelfinale | A, B, C, D, E, F, G, H |
-   | Viertelfinale | I, J, K, L & R, S, T, U |
-   | Halbfinale | M, N & V, W, X, Y, Z, AA |
-   | Finale - Woche 1 | O & AB, AC, AD, AE |
-   | Finale - Woche 2 | P, Q |
+| Stage | Match ID |
+| :-- | :-- |
+| Achtelfinale | A, B, C, D, E, F, G, H |
+| Viertelfinale | I, J, K, L & R, S, T, U |
+| Halbfinale | M, N & V, W, X, Y, Z, AA |
+| Finale - Woche 1 | O & AB, AC, AD, AE |
+| Finale - Woche 2 | P, Q |
 
 7. **Bedingungen, um zu gewinnen:**
    - In der Gruppenphase werden 4 gewonnene Maps benötigt, um die Partie zu gewinnen. (Best-of-7)
```

Comment:
I can't edit this myself, but the lines 480 to 483 should use Beatmaps instead of Maps.

Reply:
fixed